src/main/java/seedu/taassist/model/student/Student.java

```diff
@@ -71,6 +71,13 @@ public List<ModuleClass> getModuleClasses() {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Returns true if the Student is assigned to the provided {@code moduleClass}.
+     */
+    public boolean isInModuleClass(ModuleClass moduleClass) {
+        return moduleDataList.containsMatch(moduleData -> moduleData.getModuleClass().isSame(moduleClass));
```

Comment:
Optionally, we can use the `UniqueList::contains` method instead? We'll just have to construct a `ModuleDate` around it. This should make the code follow the Law of Demeter more closely.

Reply:
Implemented your suggestion 👍